aws-samples/aws-ambit-scenario-designer-ue4
# Expand and organize User Guide

## What was the problem/requirement? (What/Why)

Improve the User Guide by filling documentation gaps and improving doc organization.

## Out of scope

- Grammar, spelling, and cleanliness improvements to existing User Guide pages.

## What was the solution? (How)

- Added documentation for City Generation tools.
- Migrated setup instructions from README to User Guide.
- Split out the setup instructions for packaged Ambit plugin users and the setup instructions for developers wishing to compile from source.
- Improved setup instructions.
- Improved TOC.

## What artifacts are related to this change?

* Issue IDs: internal P53521754, P54511899

## What is the impact of this change? (Focus on the customer experience)

- Customers using the packaged Ambit plugin now have setup instructions catered to them.
- Customers wishing to compile Ambit from source now have setup instructions catered to them.
- Customers can now learn how to use the City Generation tools.

## Are you adding any new dependencies to the system?

No

## How were these changes tested?

Manually tested by running the `mkdocs serve` command and viewing the generated User Guide in a browser. 

## How to review/test

I recommend reviewers also use the approach above and review the generated docs in the browser so that formatting and images can be reviewed as the user will see them.

## Key files

The following are NEW doc pages or pages that had significant changes. Review these with special scrutiny...

- README.md
- User Guide:
  - appendix-contributing.md
  - building-generator.md
  - city-generation.md
  - obtaining-osm.md
  - road-generator.md
  - setup.md

## How does this commit make you feel? (Optional, but encouraged)

I'm passionate about good user documentation. Happy to be insisting on the highest standards for our docs.


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
